Bottom: 039938181ed773367fb1a1d7f84f745b5cb868b5
Top:    2944231e6bb331aa535adf5345c56ed6440ca820
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-07-17 14:43:31 +0800

Refresh of debug-0

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 042030e..b5da7b5 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -2271,8 +2271,11 @@ static bool vhost_notify(struct vhost_dev *dev, struct vhost_virtqueue *vq)
 	 * 1) cached used event is ahead of new
 	 * 2) old to new updating does not cross cached used event. */
 	if (vring_need_event(vq->last_used_event, new + vq->num, new) &&
-	    !vring_need_event(vq->last_used_event, new, old))
+	    !vring_need_event(vq->last_used_event, new, old)) {
+		printk("old %x new %x num %x event %x\n",
+			old, new, vq->num, vq->last_used_event);
 		return false;
+	}
 
 	/* Flush out used index updates. This is paired
 	 * with the barrier that the Guest executes when enabling
